src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -402,14 +402,29 @@ void QgsPostgresConn::unref()
   delete this;
 }
 
+/* private */
+QStringList QgsPostgresConn::supportedSpatialTypes()
+{
+  QStringList supportedSpatialTypes;
+
+  supportedSpatialTypes << quotedValue( "geometry" )
+                        << quotedValue( "geography" )
+                        << quotedValue( "pcpatch" );
+  if ( hasRaster() ) supportedSpatialTypes << quotedValue( "raster" );
+  if ( hasTopology() ) supportedSpatialTypes << quotedValue( "topogeometry" );
```

Comment:
If you want to follow the normally-used style, line-break after if clauses. Aka `astyle mkhun`